controllers/wallet.controller.js

```diff
@@ -172,97 +170,80 @@ exports.userBalance = async (req, res) => {
 }
 
 exports.getGasPrice = async (req, res) => {
-    let network = req.params.network.toUpperCase()
-    if (network === 'TRON') {
-        let tronWeb = await webTronInstance()
+    const network = req.params.network.toUpperCase()
+
+    switch (network) {
+        case 'POLYGON':
+            return exports.gasPricePolygon(req, res)
```

Comment:
I've noticed that we're using the `exports` keyword in this function, but it seems like we might not need it in this context. If this function is being used within this module and not being imported elsewhere, we can safely remove the exports keyword. This will keep the code cleaner and more readable. 
Please consider this modification.